src/test/scala/rx/lang/scala/observables/AsyncOnSubscribeTests.scala

```diff
@@ -0,0 +1,65 @@
+package rx.lang.scala.observables
+
+import org.junit.Test
+import org.junit.Assert._
+import org.scalatest.junit.JUnitSuite
+import rx.lang.scala.observers.TestSubscriber
+import rx.lang.scala.{Notification, Observable}
+
+class AsyncOnSubscribeTests extends JUnitSuite {
+
+  @Test
+  def testStateful(): Unit = {
+    val last = 2000L
+    val sut = Observable.create(AsyncOnSubscribe(() => 0L)((count,demand) =>
```

Comment:
Why `sut`? Could you please use a generally understandable name or abbreviation?